### ISSUE TITLE
Document select usage for collections

### ISSUE THREAD

**Issue by wsalembi** [belgif/rest-guide#73]
select use case is described for Document resource but I think it also applies to Collection resources

**Comment by pvdbosch** [comment]
I agree. Maybe move to Performance chapter, next to "embedding resources"?

**Comment by pvdbosch** [comment]
Would the `select` query param on collections apply to the properties in the representation in "embedded" or to "items"?
